packages/react-test-renderer/shallow.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  throw Error('shallow renderer is not available in production mode.');
+} else {
```

Comment:
> Or should we create prod versions of all of them?

I don't see the value of this, and it would cause problems with `__DEV__` only things like the non-enumerable `_owner` attribute.